client/product_details/ProductDetails.jsx

```diff
@@ -69,20 +73,47 @@ class ProductDetails extends React.Component {
 
   handleStyleSelection(e) {
     e.preventDefault();
-    console.log(e.target.id);
     var index = Number(e.target.id);
     this.setState({
       indexStyleSelected: index,
       selectedStyle: this.state.styleInfo[index],
     });
   }
 
+  handlePhotoSelection(e) {
+    e.preventDefault();
+    var index = Number(e.target.id);
+    this.setState({
+      currPhotoIndex: index
+    });
+  }
+
+  handleUpClick(e) {
+    e.preventDefault();
```

Comment:
For line 92 and 101 - do you need to prevent the default behavior? I thought it was only necessary with forms to prevent the default submit behavior, wondering if functions still work without it.

Reply:
No, I guess I don't -- I initially had an href link and clicking was leading to that image popping up in another tab. But I removed it, so no need for prevent default now. Will test this before removing on the next PR